backend/migrations/versions/44b32661fe9b_.py

```diff
@@ -0,0 +1,68 @@
+"""empty message
+
+Revision ID: 44b32661fe9b
+Revises: bd3858beccdd
+Create Date: 2020-07-30 23:47:38.624498
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '44b32661fe9b'
+down_revision = 'bd3858beccdd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Why are there so many migrations here for the events table? I feel like these are @codyxzeng's changes